apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-08-02T22:37:09Z"
+  build_date: "2021-08-03T01:38:05Z"
   build_hash: 8b9ae5dfab674f9d759bf6ccb1aef88bc91f2e4f
   go_version: go1.15.6 linux/amd64
   version: v0.8.0
-api_directory_checksum: f3302d83fa5db4156550a01a1d76ba8e3509386b
+api_directory_checksum: a204a5150e6c86b28b6da5d5ffbe934b0aa11018
 api_version: v1alpha1
 aws_sdk_go_version: v1.37.4
```

Comment:
should this be 1.37.10?

Reply:
I've pinned S3 to `1.37.4` for now - https://github.com/aws-controllers-k8s/s3-controller/blob/main/go.mod#L7

Although I wouldn't expect there to be any changes in the API between this and `1.37.10`, so probably safe to update.